magenta-lib/src/main/scala/magenta/tasks/UpdateCloudFormationTask.scala

```diff
@@ -21,6 +22,14 @@ object UpdateCloudFormationTask {
   case object UseExistingValue extends ParameterValue
   case class TemplateParameter(key:String, default:Boolean)
 
+  sealed trait CloudFormationStackLookupStrategy
+  case class LookupByName(cloudFormationStackName: String) extends CloudFormationStackLookupStrategy {
+    override def toString = s"called $cloudFormationStackName"
+  }
+  case class LookupByTags(tags: Map[String, List[String]]) extends CloudFormationStackLookupStrategy {
```

Comment:
Because with the json representation `pkgApps.size` can be be `>1`.  It would definitely be simpler to change to `Map[String,String]` if YAML enforces the size, and we are happy to just use the first app for json deploys.